src/serlo-editor-repo/plugin-text/components/text-editor.tsx

```diff
@@ -358,32 +377,28 @@ export function TextEditor(props: TextEditorProps) {
         value={state.value.value}
         onChange={handleEditorChange}
       >
-        {editable && focused && (
-          <HoveringToolbar
-            editor={editor}
-            config={config}
-            controls={toolbarControls}
-            focused={focused}
-          />
-        )}
-
-        {editable && focused && (
-          <LinkControls
-            hasSelectionChanged={hasSelectionChanged}
-            editor={editor}
-            config={config}
-            isLinkNewlyCreated={isLinkNewlyCreated}
-            setIsLinkNewlyCreated={setIsLinkNewlyCreated}
-          />
-        )}
-
         <Editable
           placeholder={config.placeholder}
           onKeyDown={handleEditableKeyDown}
           onPaste={handleEditablePaste}
           renderElement={renderElementWithEditorContext(config, focused)}
           renderLeaf={renderLeaf}
         />
+        {editable && focused && (
+          <>
+            <LinkControls
+              selectionChanged={selectionChanged}
+              editor={editor}
+              config={config}
+            />
+            <HoveringToolbar
+              editor={editor}
+              config={config}
+              controls={toolbarControls}
+              focused={focused}
+            />
+          </>
+        )}
```

Comment:
Just out of curiosity, what does moving this block after the `Editable` do?

Reply:
Before: 
When using `Tab` while the link popover is open (but not focused) the focus went to the settings button.
After 15 minutes of digging I found out that `Shift+Tab` would have set the focus to the link popover.

After:
Now when using `Tab` it moves the focus to the popover as expected. (Without needing JS to control the focus, instead relying on the normal tab order)